app/scripts/modules/core/src/cluster/filter/FilterSearch.tsx

```diff
@@ -5,27 +5,25 @@ import './FilterSearch.less';
 
 export interface IFilterSearchProps {
   helpKey?: string;
-  onBlur: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onBlur: (event: React.FocusEvent<HTMLInputElement>) => void;
   onSearchChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
   value: string;
 }
 
-export const FilterSearch = ({ helpKey, onBlur, onSearchChange, value }: IFilterSearchProps) => {
-  return (
-    <div className="filter-search">
-      <form className="horizontal middle space-between" role="form">
-        <div className="form-group nav-search">
-          <input
-            type="search"
-            className="form-control input-med sp-form"
-            value={value}
-            onBlur={onBlur}
-            onChange={onSearchChange}
-            placeholder="Search"
-          />
-        </div>
-        {helpKey && <HelpField id={helpKey} placement="right" />}
-      </form>
-    </div>
-  );
-};
+export const FilterSearch = ({ helpKey, onBlur, onSearchChange, value }: IFilterSearchProps) => (
+  <div className="filter-search sp-margin-s-right">
```

Comment:
as a convention thing, I've been naming the CSS classes that create a component scope the same as the component itself, i.e., `className="FilterSearch"`.  My reasoning is that it's easy to scan for in the DOM and indicates the root element of a component.